Code/Tools/AssetProcessor/native/utilities/UuidManager.h

```diff
@@ -0,0 +1,112 @@
+/*
```

Comment:
We don't want generation of UUIDs & meta data files outside the Asset Processor, correct? I assume that's why you put these files into the Asset Processor directly, instead of the tools framework.

Reply:
Sort of.  This class has a cache and is relying on the file monitor to clear it, and the file monitor only exists in AP.  I think we probably want to expose an API for requesting UUIDs (and probably creating them) at the tools level but I'm not sure what the design for that will be yet, it might either just not use a cache or maybe it'll interface with the AP over network messages.